src/utils/PrivateRoute.js

```diff
@@ -1,15 +1,30 @@
 import React, { useContext } from "react";
-import { Redirect, Route } from "react-router-dom";
+import { Redirect, Route, Link } from "react-router-dom";
 
 import { IsAuthContext } from "./AuthContext";
 
 function PrivateRoute({ children, ...restOfProps }) {
-  const isAuth = true
-  
-  return isAuth ? 
-    (<Route {...restOfProps}>{children}</Route>)
-       :
-    (<Redirect exact to="/" /> )
+  const { isLoggedIn } = useContext(IsAuthContext);
+  const { isAuth } = useContext(IsAuthContext);
+
+  isLoggedIn();
+
+  if (!isAuth) {
+    return (
+      <p>
+        You need to be logged in. Please login{" "}
+        <Link exact to="/">
+          here
+        </Link>
+      </p>
+    );
+  }
+
+  return isAuth ? (
+    <Route {...restOfProps}>{children}</Route>
+  ) : (
+    <Redirect exact to="/" />
```

Comment:
Shouldn't it redirect to the login page?

Reply:
Home page is the login page....